README.md

```diff
@@ -58,6 +58,22 @@ npm install -g pin-github-action
 pin-github-action /path/to/.github/workflows/your-name.yml
 ```
 
+Options are:
+```bash
+pin-github-action -h
+Usage: pin-github-action [options] [file ...]
+
+Options:
+  -V, --version                  output the version number
+  -a, --allow <actions>          comma separated list of actions to allow e.g. mheap/debug-action. May be a glob e.g. mheap/*
+  -i, --ignore-shas              do not update any commits that are pinned at a sha
+  -e, --allow-empty              allow workflows that do not contain any actions
+  -l, --yaml-line-width <width>  set maximum output width before a line break (default: "120")
+  -n, --yaml-null-str <string>   set string representation for null values (default: "null")
+  -c, --comment <string>         comment to add inline when pinning an action (default: " pin@{ref}")
+  -h, --help                     display help for command
+```
```

Comment:
```suggestion
To see all available options, run `pin-github-action --help`
```

Command lists get out of date very quickly in my experience. Would you be open to changing this to show the `--help` flag instead?

Reply:
I guess I was trying to gauge what settings I could make and see if/how this repo could be useful to me (before I installed and tried it out). 

Hence why having the list of available options would've helped in my case. 

I really understand what you mean, and maybe it's okay to have an outdated `--help` readme example, over missing this interesting info. Makes it easier to skim the readme imho.

I don't have strong feelings about this, also open to closing the PR if you don't feel this is needed. Just thought I'd contribute back a solution to a problem I faced when trying to use it. Feel the issue on maintainability though, so I'll leave it up to you, whichever is fine with me.